package.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
`package.json` (and `package-lock.json`) can be removed I think (should only live in the `gulp-project`)